modules/security-osp-validating-certificates-manually.adoc

```diff
@@ -0,0 +1,71 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_openstack/preparing-to-install-on-openstack.adoc
+
+:_content-type: PROCEDURE
+[id="security-osp-validating-certificates-manually_{context}"]
+= Scanning {rh-openstack} endpoints for legacy HTTPS certificates manually
+
+Beginning with {product-title} 4.10, HTTPS certificates must contain subject alternative name (SAN) fields. If you do not have access to the prerequisite tools that are listed in "Scanning {rh-openstack} endpoints for legacy HTTPS certificates", perform the following steps to scan each HTTPS endpoint in a {rh-openstack-first} catalog for legacy certificates that only contain the `CommonName` field.
+
+[IMPORTANT]
+{product-title} does not check the underlying {rh-openstack} infrastructure for legacy certificates prior to installation or updates. Use the following steps to check for these certificates yourself. Failing to update legacy certificates prior to installing or updating a cluster will result in cluster dysfunction.
+
+.Procedure
+
+. On a command line, run the following command to view the URL of {rh-openstack} public endpoints
++
+[source,terminal]
+----
+$ openstack catalog list
+---- 
++
+Record the URL for each HTTPS endpoint that the command returns.
+. For each endpoint, record the host and the port. 
```

Comment:
The above command shows all endpoints. Here this could be: "for each _public_ endpoint, record the host and the port

Reply:
```suggestion
. For each public endpoint, record the host and the port. 
```